File: PT-Charlie/JS_Challenges/99-Bottles/99-bottles.js
function bottleSong() {
    
    var word = "bottles";
    var count = 99;
    while (count > 0) {
        var bottle = count == 1 ? "bottle" : "bottles";
        console.log(count + " " + word + " of beer on the wall");
        console.log(count + " " + word + " of beer,");
        console.log("Take one down, pass it around,");
        --count;
        var bottle = count == 1 ? "bottle" : "bottles";
        console.log(count + " " + word + " of beer on the wall.");
    } 
}
function bottleSong() { 

    var bottles;
    for (counter = 99; counter >= 1; counter = counter - 1) 
    {
        if (counter == 1) {
            bottles = 'bottle';
        } else {
            bottles = 'bottles';
        }
        console.log(counter+" "+bottles+" of beer on the wall.");
        if (counter < 99) {
            console.log("");
            console.log(counter+" "+bottles+" of beer on the wall.");
        }
        console.log(counter+" "+bottles+" of beer.");
        console.log("Take one down.");
        console.log("Pass it around.");
        if (counter == 1) {
            console.log("No bottles of beer on the wall.");
        }
    }
}

bottleSong()

File: PT-Charlie/JS_Challenges/99-Bottles/99-bottles.py
def bottle_song(num_bottles):
    
    if(num_bottles == 1):
        objects = 'bottle'
        object_minus_one = 'bottle'
    elif(num_bottles == 2):
        objects = 'bottles'
        object_minus_one = ' bottle'
    else:
        objects = 'bottles'
        object_minus_one = 'bottles'
    
    if(num_bottles > 0):
        print(str(num_bottles) + " " + objects + " of beer on the wall, " + str(num_bottles) + " " + objects + " of beer.")
        print("Take one down and pass it around, " + str(num_bottles-1) + " " + object_minus_one + " of beer on the wall.")
        print(" ")
    elif(num_bottles == 0):
        print("No more bottles of beer on the wall, no more bottles of beer.")
        print("Go to the store and buy some beer, 99 bottles of beer on the wall.")
    else:
        print("Error: Wheres the booze?")

bottles = 99
    
while bottles >= 0:
    bottle_song(bottles)
    bottles -= 1
   
